lib/watcherManager.js

```diff
@@ -19,13 +19,13 @@ WatcherManager.prototype.getDirectoryWatcher = function(directory) {
 	return this.directoryWatchers[directory];
 };
 
-WatcherManager.prototype.watchFile = function watchFile(p, startTime) {
+WatcherManager.prototype.watchFile = function watchFile(p, startTime, options) {
 	var directory = path.dirname(p);
-	return this.getDirectoryWatcher(directory).watch(p, startTime);
+	return this.getDirectoryWatcher(directory).watch(p, startTime, options);
```

Comment:
should be `return this.getDirectoryWatcher(directory, options).watch(p, startTime);` instead.
same for `watchDirectory`